Formula/gmssl.rb

```diff
@@ -0,0 +1,22 @@
+class Gmssl < Formula
+  desc "Toolkit for Chinese national cryptographic standards"
+  homepage "https://github.com/guanzhi/GmSSL"
```

Comment:
interesting enough the site is not accessible
![image](https://user-images.githubusercontent.com/1580956/222539676-8e90770b-435d-4199-9297-382624912023.png)

Reply:
Yes, that's why I did not use that site as the `homepage`.

Reply:
👍